client-cli/src/command/wallet_command.rs

```diff
@@ -35,6 +35,13 @@ pub enum WalletCommand {
             case_insensitive = true
         )]
         wallet_type: WalletKind,
+        #[structopt(
+            name = "wallet mnemonic count",
+            short = "m",
```

Comment:
perhaps one more argument:
```
default_value = "24"
```

so the old commands still work as before?

Reply:
yes, old commands is the same.
i'll add default value.